scripts/sanity_chk/sanitylib.py

```diff
@@ -1617,6 +1617,7 @@ def create_overlay(self, platform, enable_asan=False, enable_coverage=False, cov
                     content = content + "\nCONFIG_ASAN=y"
 
             f.write(content)
+            return content
 
```

Comment:
this should be a separate  commit.

Reply:
you could still have this as part of the same PR, but it should be in its own commit, because it is not related to the test directly.